.gitignore

```diff
@@ -106,7 +106,7 @@ docs/_build/
 
 # Go
 iroh-node-go
-/iroh-go/iroh/ffi/
+/iroh-go/iroh/libiroh.a
```

Comment:
I think there is a different file extension on windows?

Reply:
adjusted it, but also planning to do a mini push for windows after this PR, now that i have access to one again.